app/src/service_endpoints.h

```diff
@@ -172,9 +230,84 @@ namespace scitt
     auto service_certificate_index =
       std::make_shared<ServiceCertificateIndexingStrategy>();
 
+    auto service_key_index = std::make_shared<ServiceKeyIndexingStrategy>();
+
     context.get_indexing_strategies().install_strategy(
       service_certificate_index);
 
+    context.get_indexing_strategies().install_strategy(service_key_index);
+
+    auto get_transparency_config =
+      [&](ccf::endpoints::ReadOnlyEndpointContext& ctx) {
+        auto subsystem =
+          context.get_subsystem<ccf::cose::AbstractCOSESignaturesConfig>();
+        if (!subsystem)
+        {
+          ctx.rpc_ctx->set_error(
+            HTTP_STATUS_INTERNAL_SERVER_ERROR,
+            ccf::errors::InternalError,
+            "COSE signatures subsystem not available");
+          return;
+        }
+        auto cfg = subsystem->get_cose_signatures_config();
+
+        nlohmann::json config;
+        config["issuer"] = cfg.issuer;
+        config["jwksUri"] = fmt::format("https://{}/jwks", cfg.issuer);
+
+        const auto accept =
+          ctx.rpc_ctx->get_request_header(ccf::http::headers::ACCEPT);
+        if (accept.has_value())
+        {
+          const auto accept_options =
+            ccf::http::parse_accept_header(accept.value());
+          if (accept_options.empty())
+          {
+            throw ccf::RpcException(
+              HTTP_STATUS_NOT_ACCEPTABLE,
+              ccf::errors::UnsupportedContentType,
+              fmt::format(
+                "No supported content type in accept header: {}\nOnly {} is "
+                "currently supported",
+                accept.value(),
+                ccf::http::headervalues::contenttype::JSON));
```

Comment:
This is parsing the Accept header sent by a client in their GET request, and this case is specifically dealing with something like `Accept: `, or `Accept: ,`, where the header is set, but the value is empty or does not contain any valid content type or expression (something like `*/*` is ok).

`application/json` and `cbor`, as well as `*/*` and `application/*` are accepted and handled below.

We could change the behaviour on empty Accept, to be the same as no Accept at all (line 302), but it seemed best to be strict and reject, because it's probably a malformed request? I don't have a strong opinion.

Reply:
No, I mean the error says only json is supported, this is what would be confusing in such situation `No supported content type in accept header: \nOnly application/json is currently supported`

Reply:
Ah yes, I'd missed that, thank you.